tests/sentry/integrations/discord/test_client.py

```diff
@@ -0,0 +1,69 @@
+import responses
+
+from sentry import options
+from sentry.integrations.discord.client import DiscordClient
+from sentry.shared_integrations.exceptions import IntegrationError
+from sentry.testutils.cases import TestCase
+
+
+class DiscordClientTest(TestCase):
```

Comment:
It'll be nice to have some proxy tests. See here for an example: https://github.com/getsentry/sentry/blob/6eeda705f3d17da16538038136b27c3ec8afbd57/tests/sentry/integrations/github/test_client.py#L345-L349